packages/dataviews/src/constants.ts

```diff
@@ -69,24 +56,3 @@ export const SORTING_DIRECTIONS = {
 export const LAYOUT_TABLE = 'table';
 export const LAYOUT_GRID = 'grid';
 export const LAYOUT_LIST = 'list';
-
-export const VIEW_LAYOUTS = [
```

Comment:
I moved this out of this file because it would have required typing all the components and I needed the constants file to be typed.